README.md

```diff
@@ -13,6 +13,32 @@ This project is currently built with and tested against Hive 2.3.0 (and minor ve
 # Usage
 The BeeJU JUnit rules provide a way to run tests that have an underlying requirement to use the Hive Metastore API but don't have the ability to mock the [Hive Metastore Client](https://hive.apache.org/javadocs/r1.2.1/api/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.html). The rules spin up and tear down an in-memory Metastore which may add few seconds to the test life cycle so if you require tests to run in the sub-second range this is not for you.
 
+##Maven Dependencies
```

Comment:
You need spaces after the ## - take a look at https://github.com/HotelsDotCom/beeju/tree/upgrade-to-junit5 and see how it renders.

Reply:
heh, i'd blame that on IntelliJ, but then you just have more reasons... :))

Reply:
Yes ;) Ultimately the only MD rendering engine we care about here is GitHub's so I always check what it looks like there. This is one of the big drawbacks of MD is that there are so many different ways to interpret it :(